source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -70,195 +83,220 @@ Mixing
    want to add your own RPMs to the Mix. If you are working only with Clear
    bundles, then skip to Step 7.)
 
    If you are creating RPMs from scratch, you may use ``autospec``,
-   ``mock``, ``rpmbuild``, etc. to build them. If they are not
-   built on Clear, make sure your configuration and toolchain builds them correctly for Clear, or there is no guarantee
-   they will be compatible.
+   ``mock``, ``rpmbuild``, etc. to build them. If they are not built on Clear,
+   make sure your configuration and toolchain builds them correctly for Clear,
+   or there is no guarantee they will be compatible.
+
+#. **Import RPMs into workspace**. Create an :file:`rpms` directory in your
+   workspace (for example :file:`/home/clr/mix/rpms`), and copy the RPMs you
+   want into that directory. Next, add the following to your
+   :file:`builder.conf`::
 
-#. **Import RPMs into workspace**. The way to do this is to create an
-   ``rpms`` directory in your workspace (for example ``/home/clr/mix/rpms``),
-   and to copy the RPMs you want into that directory. The mixer script will
-   look here for RPMs in order to build a local RPM repo for yum to use.
+      RPMDIR=/home/clr/mix/rpms
+
+   Mixer will look here for RPMs in order to build a local RPM repo for yum to
+   use.
 
 #. **Create a local RPM repo**. Create an empty directory in your workspace
-   named ``local`` and add the paths in your builder.conf::
+   named :file:`local` and add the path in your :file:`builder.conf`::
 
-    RPMDIR=/home/clr/mix/rpms
-    REPODIR=/home/clr/mix/local
+      REPODIR=/home/clr/mix/local
 
-    These variables are automatically read; you simply need to run::
+   Once these values are configured, you can generate the yum repo by running::
 
       # sudo mixer add-rpms
 
-   After the script exits, you should see your RPMs and a repodata directory in
-   ``/home/clr/mix/local``. If the RPMs are not all in the local directory, check
-   to make sure that they are indeed valid RPM files and not corrupt.
+   After the tool exits, you should see your RPMs and a repodata directory in
+   :file:`/home/clr/mix/local`. If the RPMs are not all in this :file:`local`
+   directory, check to make sure that they are indeed valid RPM files and not
+   corrupt.
+
+#. **Update/Add bundle definitions**. You can easily add bundles to your mix by
+   running::
 
-#. **Update/Add bundle definitions**. The mixer uses a local clone of the
-   ``clr-bundles`` repo to define bundles for the mix.
+      # sudo mixer bundle add bundle1,bundle2,...
+
+   This command will copy the specified bundle defintion files from your
+   configured upstream version of Clear Linux (:file:`.clearversion`) into your
+   :file:`mix-bundles` directory.
+
+   Behind the scenes, mixer uses a local cache of the upstream Clear Linux
+   bundle definitions. These are stored in the 
+   :file:`.mixer/upstream-bundles/clr-bundles-{VER}/bundles/` directory in your
```

Comment:
@kevincwells I'm sure this doesn't need to be said, but this kind of content will have to be updated as the new bundle/folder PR changes go in soon :)